src/order/sup_indep.lean

```diff
@@ -16,16 +16,15 @@ In distributive lattices, this is equivalent to being pairwise disjoint.
 
 ## Implementation notes
```

Comment:
Could you add `## Main declarations`?

Reply:
Done, did you want any others in the list?

Reply:
No, it's perfect!